addons/website_slides_survey/views/website_profile.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo><data>
+    <template id="user_profile_content" inherit_id="website_profile.user_profile_content">
+        <xpath expr="//div[@id='profile_about_badge']" position="before">
+            <t t-if="channel">
+                <div class="mb32">
+                    <h4><i class="fa fa-trophy"/> Certifications</h4>
+                    <t t-call="website_slides_survey.display_certificate"/>
+                </div>
+            </t>
+        </xpath>
+    </template>
+
+     <template id="display_certificate">
+        <t t-if="certificates">
+            <div class="row">
+                <div class="col-lg-6 o_wprofile_slides_card" t-foreach="certificates" t-as="certificate">
+                    <div class="card-body o_wprofile_slides_certif_card_body o_wprofile_bb_grey">
```

Comment:
Note for later: lot of custom css / classes for cards. It should probably be standardized for all profile page.